modules/ROOT/pages/clustering/clustering-advanced/unbind.adoc

```diff
@@ -12,7 +12,9 @@ Therefore, it is strongly recommended to consult with Neo4j Support before using
 
 Use the `unbind` command only when troubleshooting **a specific server** and remember there is no guarantee that the allocator will reassign the same databases to this server, potentially resulting in orphaned database stores.
 
-The `unbind` command preserves all database stores on the server; and when the unbound server is restarted and enabled, it is seen as an entirely new server.
+The `unbind` command preserves all database stores on the server; and when the unbound server is restarted, it is seen as an entirely new server - so it won't host any of the database it did host before the operation.
```

Comment:
```suggestion
The `unbind` command preserves all database stores on the server; and when the unbound server is restarted, it is seen as an entirely new server.
Therefore, it will not host any of the databases it hosted before the operation.
```

Reply:
Should we add here that you can manually re-attach databases to the server? Database stores are still there.